backend/app/views/spree/admin/style_guide/topics/typography/_colors.html.erb

```diff
@@ -0,0 +1,64 @@
+<%- snippet = capture do %>
+  <p>where the html example goes</p>
+<%- end %>
+
+<p>
+  The Solidus UI is made up mostly of a grey color palette with blue being used for interactive elements and red, green and yellow being used to give visual context to messaging.
+</p>
+
+<%-
+colors = [
```

Comment:
That's a neat idea. A related problem comes up when trying to communicate styles (like media query breakpoints) between css and js. I've experimented with using ruby and the canonical source and templating into assets, but everyone seems to get all up in arms about the fanciness.